### ISSUE TITLE
object-literal-key-quotes false positive

### ISSUE THREAD

**Issue by anvish** [Microsoft/tslint-microsoft-contrib#343]
The rule suggests removing quotes around empty string
```ts
const a = {
    '': 'b', // [tslint] Unnecessarily quoted property '' found. (object-literal-key-quotes)
}
```

**Comment by HamletDRC** [comment]
Hi @anvish 
The object-literal-key-quotes rule is a base TSLint rule. 

Can you please create this defect on that project? It is not my source code :(
https://github.com/palantir/tslint/issues/new

**Comment by anvish** [comment]
Seems to be fixed there in latest version.
